jupyterlab_caip_optimizer/src/components/create_study.spec.tsx

```diff
@@ -33,3 +39,315 @@ describe('Dropdown list', () => {
     expect(dropdown).toEqual(goalDropdownList);
   });
 });
+
+describe('Create Study Page', () => {
+  it('creates a study and redirects to the study details page', async () => {
```

Comment:
This is really strange. I am pretty sure that these tests are already on master.